### PR TITLE
⚪  Add Circle icon

### PR DESCRIPTION
**Pull request by code-arman**
## Status:
:rocket: Ready

## Description

Creates a CircleIcon component which takes in a child (either an icon or img), and styles it correctly.

## Screenshots
Screen Shot 2021-03-13 at 7 56 06 PM
Screen Shot 2021-03-13 at 7 55 43 PM

### CODE REVIEW

File: src/components/ui/ListEntry.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ const Icons = styled.div`
   align-items: center;
   max-height: 36px;
   & > * {
-    padding: 8px;
+    margin: 8px;
```
Comment (Qrtn) [inline src/components/ui/ListEntry.js:11]: Should this be horizontal margin only?

**Review by Qrtn — APPROVED** [review]
Otherwise, LGTM, beautiful work 🔵

File: src/components/ui/ListEntry.js (hunk 1 above)
Comment (Qrtn) [reply src/components/ui/ListEntry.js:11]: Also it looks like a lot of margin in the screenshot, double-check that with Figma.

**Review by laurenho025 — APPROVED** [review]
eh looks like it should be fine

**Comment by code-arman** [comment]
> eh looks like it should be fine

thx @n3a9 🥰

File: src/components/ui/ListEntry.js (hunk 1 above)
Comment (code-arman) [reply src/components/ui/ListEntry.js]: Good point. Distance looks too big

File: src/components/ui/CircleIcon.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  overflow: hidden;
+
+  ${StyledIconBase} {
+    width: 65%;
+    color: #fff;
+  }
+  & > img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
```
Comment (SirajChokshi) [inline src/components/ui/CircleIcon.js:23]: 🤑

File: src/components/ui/ListEntry.js [hunk 3]
```diff
@@ -8,7 +8,7 @@ const Icons = styled.div`
   align-items: center;
   max-height: 36px;
   & > * {
-    padding: 8px;
+    margin: 0 4px;
```
Comment (SirajChokshi) [inline src/components/ui/ListEntry.js:11]: We can try adding a selector like `&:not(:last-child)` to add a different margin to the last element here

**Review by SirajChokshi — APPROVED** [review]
Looks good!

File: src/components/ui/ListEntry.js (hunk 3 above)
Comment (code-arman) [reply src/components/ui/ListEntry.js]: worked amazingly 😍

**Pull request merged**